src/main/java/net/ltxprogrammer/changed/entity/beast/PooltoyWolf.java

```diff
@@ -20,8 +23,9 @@ public PooltoyWolf(EntityType<? extends PooltoyWolf> p_19870_, Level p_19871_) {
 
     @Override
     protected void setAttributes(AttributeMap attributes) {
-        super.setAttributes(attributes);
-        AttributePresets.wolfLike(attributes);
+        attributes.getInstance(Attributes.MOVEMENT_SPEED).setBaseValue(0.2);
```

Comment:
Why was the movement speed set so low?

Reply:
oops, was tinkering around with values, i'll revert that